src/axom/inlet/docs/sphinx/advanced_types.rst

```diff
@@ -40,16 +54,52 @@ schema on that subtable, e.g.:
 
 The definition of a static ``defineSchema`` member function is not required, and is just used
 for convenience.  The schema definition for a class or struct could also be implemented as a
-free function for third-party types, or even in the same place as the sub-table declaration. 
+free function for third-party types, or even in the same place as the sub-table declaration.
 
-Arrays of user-defined types are also supported in Inlet.  First, use the ``addStructArray``
-function to create a subtable, then define the schema on that table:
+Accessing
+---------
+
+In order to have Inlet extract from the input file into a struct, it needs to know how that struct can be built.
+This is accomplished by a specializing of the struct ``FromInlet<T>`` for your type ``T``.  It must define a
+single member function with the signature ``T operator()(const inlet::Table&)``.  This function should return
```

Comment:
> with the corresponding fields in the \*input file\*.

I still have issues with "input file" here, since it is populated from the in-memory datastore representation after it is read in.  The values can be modified during parsing (e.g. via a default value) or later (e.g. by the user)